pallet/account-migration/src/lib.rs

```diff
@@ -173,8 +185,157 @@ pub mod pallet {
 		) -> DispatchResult {
 			ensure_none(origin)?;
 
+			Self::migrate_inner(from, to)?;
+
+			Ok(())
+		}
+
+		/// Similar to `migrate` but for multisig accounts.
+		///
+		/// The `_signature` should be provided by `who`.
+		#[pallet::call_index(1)]
+		#[pallet::weight(0)]
+		pub fn migrate_multisig(
+			origin: OriginFor<T>,
+			who: AccountId32,
+			others: Vec<AccountId32>,
+			threshold: u16,
+			to: AccountId20,
+			_signature: Signature,
+		) -> DispatchResult {
+			ensure_none(origin)?;
+
+			let (members, multisig) = multisig_of(who, others, threshold);
+
+			if threshold < 2 {
+				Self::migrate_inner(multisig, to)?;
+			} else {
+				let mut members = members.into_iter().map(|m| (m, false)).collect::<Vec<_>>();
+
+				// Set the status to `true`.
+				//
+				// Because the `_signature` was already been verified in `pre_dispatch`.
+				members
+					.last_mut()
+					.expect("[pallet::account-migration] `members` will never be empty; qed")
+					.1 = true;
```

Comment:
Do we need to `threshold + 1`  after updating the member's status?

Reply:
Threshold is a fixed value.

e.g. `3/5` is always `3/5`.